lib/postgres.dart

```diff
@@ -364,13 +365,22 @@ final class Endpoint {
   final String? password;
   final bool isUnixSocket;
 
+  /// The [SecurityContext] to use when connecting to this endpoint.
+  ///
+  /// This can be configured to only allow some certificates. When used,
+  /// [ConnectionSettings.sslMode] should be set to [SslMode.verifyFull], as
+  /// this package will allow other certificates or insecure connections
+  /// otherwise.
+  final SecurityContext? securityContext;
```

Comment:
I'm wondering: since `sslMode` is already in the `ConnectionSettings`, why not put this there too?

Reply:
My idea was that, for pools, one might want to connect to two endpoints potentially using different certificate chains. But that looks like a very specific use case, so I'm fine with moving this to `ConnectionSettings` as well.